addons/account/tests/test_account_move_entry.py

```diff
@@ -656,3 +656,16 @@ def test_cache_invalidation(self):
         self.assertEqual(self._get_cache_count(), 1)
         self.env['account.move.line'].invalidate_cache(ids=lines.ids)
         self.assertEqual(self._get_cache_count(), 0)
+
+    def test_misc_prevent_edit_tax_on_posted_moves(self):
+        # You cannot remove journal items if the related journal entry is posted.
+        self.test_move.action_post()
+        with self.assertRaises(UserError), self.cr.savepoint():
```

Comment:
What errors are you expecting?
I wouldn't be surprised if many errors could be raised for different reasons.
Unbalanced move? Tax lock? Cannot changed posted item?

Maybe check the message of the error?